test/fixtures/parser/tsql/create_table_with_sequence.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE DEST.MELDER(
+      [ID] INT PRIMARY KEY NOT NULL DEFAULT NEXT VALUE FOR [dbo].[SEQ_MELDER]
+      ,[DOWNLOADED_TOTAL] INT NULL
+      ,[WARNED_DAILY] INT NULL
+      ,[DATE_OF_REPORT] DATETIME NULL
+      ,DATE_LAST_INSERTED DATETIME DEFAULT GETDATE()
+);
```

Comment:
Nit can we add newlines? We have an `.editorconfig` file if you editor is compatible:

```suggestion
);

```

There's a few `.sql` files that have this issue.